modules/installing-oadp-aws-sts.adoc

```diff
@@ -220,15 +220,15 @@ $ cat << EOF | oc create -f -
 EOF
 ----
 <1> Set this field to `false` if you do not want to use image backup.
-<2> See the first note regarding the `nodeAgent` attribute.
+<2> See the note regarding the `nodeAgent` attribute.
```

Comment:
I am not sure about this but should we mention "See the important note regarding......"?